pages/wiki/update-asteroidos.hbs

```diff
@@ -3,30 +3,29 @@ title: Update AsteroidOS
 layout: documentation
 ---
 
-<p>As currently no AsteroidOS watches have direct access to the Internet, no OTA upgrades are offered. This page documents upgrading AsteroidOS.</p>
+As currently no AsteroidOS watches have direct access to the Internet, no OTA upgrades are offered. This page documents upgrading AsteroidOS.
 
-<div class="page-header">
-  <h1 id="reflash">Reflash AsteroidOS</h1>
-</div>
 
-<p>The easiest way is to reflash the entire OS following the usual <a href="https://asteroidos.org/install/">installation instructions</a>.</p>
+#Reflash AsteroidOS
```

Comment:
In this line, you are defining a [headline](https://www.markdownguide.org/basic-syntax/#headings). Headlines should have a [space between the "#" and the text ](https://www.markdownguide.org/basic-syntax/#heading-best-practices) 

in this case: `# Reflash AsteroidOS`